### PR TITLE
Integration between student table and Student Info

### PR DESCRIPTION
**Pull request by TortoiseRevenge**

### CODE REVIEW

File: src/components/table-layout/dynamicTable.js
Comment (Jihong-Bae) [inline src/components/table-layout/dynamicTable.js:1]: you added rowId 

i used 
const { Id } = row;

**Review by Jihong-Bae — APPROVED** [review]
Approved by Jihong

**Review by abby-winn — APPROVED** [review]

**Review by brandon-moyer — APPROVED** [review]
goalstable will not be missed

**Review by omrajpal — APPROVED** [review]

**Review by hraetz — APPROVED** [review]
Good

**Pull request merged**